<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1178</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BJ Printer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_7">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="mainWindow">
       <attribute name="title">
        <string>Main Window</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_8">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item row="4" column="4">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="1">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="6">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="5">
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item row="2" column="0" colspan="2">
           <widget class="QPushButton" name="getXAxisPosition">
            <property name="text">
             <string>Get X-Axis Position</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QCheckBox" name="activateRoller1">
            <property name="text">
             <string>Activate Roller</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="3" alignment="Qt::AlignHCenter|Qt::AlignTop">
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="text">
             <string>Misc. Control</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="getYAxisPosition">
            <property name="text">
             <string>Get Y-Axis Position</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QPushButton" name="getZAxisPosition">
            <property name="text">
             <string>Get Z-Axis Position</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="7" column="1">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="1">
           <widget class="QLabel" name="X_axis">
            <property name="text">
             <string>X Axis</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="Y_axis">
            <property name="text">
             <string>Y Axis</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="VelocityLabel">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="text">
             <string>Velocity
(mm/s)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0" rowspan="3">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="3" rowspan="3">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="2">
           <widget class="QDoubleSpinBox" name="xVelocity">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="minimum">
             <double>0.010000000000000</double>
            </property>
            <property name="maximum">
             <double>150.000000000000000</double>
            </property>
            <property name="value">
             <double>30.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QDoubleSpinBox" name="yVelocity">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="minimum">
             <double>0.010000000000000</double>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
            <property name="value">
             <double>30.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="10" column="5" colspan="2">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="1" column="0" colspan="2">
           <widget class="QPushButton" name="removeBuildBox">
            <property name="text">
             <string>Position Build Box for Removal</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <spacer name="verticalSpacer_9">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QPushButton" name="stopMotionButton">
            <property name="text">
             <string>
Stop Motion
</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="3" rowspan="3">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="2" column="3">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Move to Position</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QDoubleSpinBox" name="zAbsoluteMoveSpinBox">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-15.000000000000000</double>
            </property>
            <property name="maximum">
             <double>15.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0" rowspan="5">
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>12</number>
            </property>
            <item alignment="Qt::AlignHCenter">
             <widget class="QToolButton" name="zMax">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
                <kerning>true</kerning>
               </font>
              </property>
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Move z-axis to top&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>⤊</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
             <widget class="QToolButton" name="zUp">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
                <bold>true</bold>
                <kerning>true</kerning>
               </font>
              </property>
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:400;&quot;&gt;Move z-axis up one layer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>⇧</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
             <widget class="QToolButton" name="zDown">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
                <bold>true</bold>
                <kerning>true</kerning>
               </font>
              </property>
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:400;&quot;&gt;Move z-axis down one layer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>⇩</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QToolButton" name="zMin">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
                <kerning>true</kerning>
               </font>
              </property>
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Move z-axis to bottom&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>⤋</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="5" column="3" alignment="Qt::AlignTop">
           <widget class="QPushButton" name="zAbsoluteMoveButton">
            <property name="text">
             <string>Move Z</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Z Step Size</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <spacer name="verticalSpacer_7">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0" colspan="2" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="text">
             <string>Z-Axis Control</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="zStepSize">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>15000</number>
            </property>
            <property name="value">
             <number>30</number>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>(Microns)</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <spacer name="verticalSpacer_8">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="3">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>(mm)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="4" column="2">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="1" rowspan="3">
         <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0">
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="4" column="0">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="7" column="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QToolButton" name="yHome">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="text">
             <string>Y
Home</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QToolButton" name="xHome">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="text">
             <string>X
Home</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QToolButton" name="yDownButton">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>20</pointsize>
              <bold>true</bold>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="text">
             <string>▼</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3" alignment="Qt::AlignHCenter">
           <widget class="QToolButton" name="xRightButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>30</pointsize>
              <italic>false</italic>
              <bold>true</bold>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="text">
             <string>▶</string>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonIconOnly</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QToolButton" name="yUpButton">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>20</pointsize>
              <bold>true</bold>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="text">
             <string>▲</string>
            </property>
           </widget>
          </item>
          <item row="8" column="2">
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="1" alignment="Qt::AlignHCenter">
           <widget class="QToolButton" name="xLeftButton">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>30</pointsize>
              <bold>true</bold>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="text">
             <string>◀</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="3" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="text">
             <string>X/Y Axis Control</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="9" column="1" rowspan="2">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="5" rowspan="3">
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="connect">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Click to connect to the Galil motion controller and run the homing sequence for the printer.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="statusTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="text">
             <string>
Connect to Controller
</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="homeZAxisCheckBox">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Uncheck this if it is required to keep the z-axis stationary when connecting to the controller&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Home Z-Axis</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="tristate">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="connectToJetDriveButton">
          <property name="text">
           <string>
Connect to JetDrive
</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1178</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuPrinter_Controls">
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionShow_Hide_Console"/>
    <addaction name="actionShow_Hide_Droplet_Tool"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuPrinter_Controls"/>
   <addaction name="menuEdit"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionShow_Hide_Console">
   <property name="text">
    <string>Show/Hide Console</string>
   </property>
  </action>
  <action name="actionShow_Hide_Droplet_Tool">
   <property name="text">
    <string>Show/Hide Droplet Tool</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
